### PR TITLE
Use kustomize to deploy CA operator in k8s

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

File: pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go [hunk 1]
```diff
@@ -64,4 +64,5 @@ type ScaleDownConfig struct {
 	DelayAfterAdd     string `json:"delayAfterAdd"`
 	DelayAfterDelete  string `json:"delayAfterDelete"`
 	DelayAfterFailure string `json:"delayAfterFailure"`
+	UnneededTime      string `json:"unneededTime"`
```
Comment (frobware) [inline pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go]: I think the variable should reflect `scale-down`. And it should be optional too.

File: pkg/controller/clusterautoscaler/clusterautoscaler.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ func (a AutoscalerArg) TypeRange(t string, min, max int) string {
 // operator when processing ClusterAutoscaler resources.
 const (
 	LogToStderrArg                  AutoscalerArg = "--logtostderr"
+	VerboseArg                      AutoscalerArg = "--v=3"
```
Comment (frobware) [inline pkg/controller/clusterautoscaler/clusterautoscaler.go]: I've typically found `4` to be more useful.

File: pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go (hunk 1 above)
Comment (frobware) [reply pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go]: Ah, but this is in `ScaleDownConfig`... ok, forgot my comment.

**Comment by ingvagabund** [comment]
Just verified I am able to scale down pretty fast after applying the PR.

File: pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go (hunk 1 above)
Comment (ingvagabund) [reply pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go]: > I think the variable should reflect `scale-down`

I what sense? It's under `ScaleDownConfig` struct.

> And it should be optional too.

Yeah. Agree.

File: kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Adds namespace to all resources.
+namespace: kube-system
+
+# Value of this field is prepended to the
```
Comment (frobware) [inline kustomization.yaml]: If we are not using any of these keys then why not delete them all? Without them I don't have to guess whether I need them or not.

Comment (ingvagabund) [reply kustomization.yaml]: Removed

**Comment by enxebre** [comment]
/hold
please hold until pivot is done, we have no e2e tests here so we don't want to introduce any unnecessary risk

**Pull request merged**